### ISSUE TITLE
Unkown error when trying to access AWS ParallelCluster UI

### ISSUE THREAD

**Issue by Malte1911** [aws/aws-parallelcluster-ui#348]
When trying to access the ParallelCluster UI I get the following error page after logging in: 
![image](https://github.com/user-attachments/assets/7b2e923b-538f-4d23-85f9-ec84f39a54c6)
I am deploying the standard template, with no configuration other than my admin email. I am using the newest version on us-east-2. The login itself works without problems. I already tried setting it up again yet that does not seem to change the issue. The issue persists on eu-central-1.

When clicking on "Go to Homepage", it sometimes redirects me to the expected page and shows me an internal server error. I cannot figure out what the problem is, whether it is a problem on my end or a bug.

**Comment by dreambeyondorange** [comment]
Can you tell us the steps you took to see this error so that we can reproduce it?  It seems to be a known issue but want to confirm.

**Comment by Malte1911** [comment]
Hello, I just followed this guide [here](https://docs.aws.amazon.com/parallelcluster/latest/ug/install-pcui-v3.html). There was nothing else I did except follow the tutorial to setup the UI. The login went as expected but as soon as I changed the password and went onto the main site the above error window got shown. After waiting another day to login again the same error occurred. There is sadly not that much more info I can give you since there is nothing I did other than what the guide said. If you want I can do a screen recording of me setting it up but I doubt that will casst any light on anything out of the ordinary...
Thank you a lot though for your quick response!

**Comment by dreambeyondorange** [comment]
> Hello, I just followed this guide [here](https://docs.aws.amazon.com/parallelcluster/latest/ug/install-pcui-v3.html). There was nothing else I did except follow the tutorial to setup the UI. The login went as expected but as soon as I changed the password and went onto the main site the above error window got shown. After waiting another day to login again the same error occurred. There is sadly not that much more info I can give you since there is nothing I did other than what the guide said. If you want I can do a screen recording of me setting it up but I doubt that will casst any light on anything out of the ordinary... Thank you a lot though for your quick response!

No problem, thanks for the explanation!  Does the error message prevent you from performing any tasks in the UI?

**Comment by Malte1911** [comment]
Yes. When closing it it still just show up again. As already said, when reloading I can get into the main UI, where I can do everything normally, yet I can't push anything to the cloud or do a dry-run of my cluster.

**Comment by gmarciani** [comment]
Hi @Malte1911 ,

we cannot reproduce the issue with the latest PCUI 2024.10.0.
Would moving to PCUI 2024.10.0 be an option for you?

Thanks,
Giacomo
